src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -21,9 +21,9 @@
             + PREFIX_NAME + "NAME "
             + PREFIX_EXPIRY_DATE + "EXPIRY DATE "
             + "[" + PREFIX_TAG + "TAG]...\n"
-            + "Example: " + COMMAND_WORD + " "
+            + "Example: " + COMMAND_WORD + "| "
             + PREFIX_NAME + "Strawberry "
-            + PREFIX_EXPIRY_DATE + "11/12/1999"
+            + PREFIX_EXPIRY_DATE + "11/12/1999 "
```

Comment:
Hi guys, will all the prefixes e.g. PREFIX_NAME also be changed to "|" later on? @liawsy @JermyTan

Reply:
Yes, according to the formatting on https://ay1920s1-cs2103t-f11-2.github.io/main/UserGuide.html